pkgs/by-name/in/insync-nautilus/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  stdenv,
+  fetchurl,
+  lib,
+  dpkg,
+  gnome,
+  insync
+}:
+
+stdenv.mkDerivation rec {
+  pname = "insync-nautilus";
+  version = lib.getVersion insync;
+  pyproject = true;
+
+  # Download latest from: https://www.insynchq.com/downloads/linux#nautilus
+
+  src = fetchurl {
+    url = "https://cdn.insynchq.com/builds/linux/insync-nautilus_${version}_all.deb";
```

Comment:
Usually, it's good to have 2 URLs with a https://web.archive.org/ fallback, so that the package can be built even if the upstream link fails.

You can save the URL at `https://web.archive.org/save/$url` and then set `urls = [ ... ]` instead of `url` in the `fetchurl` call

Reply:
let me try.

Reply:
I saved those in the web archive.

But I had to keep two separate fetchurl, because they were referenced separately in the `unpackPhase`.

please check again